src/components/LiveCursors/Cursors.tsx

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useState } from 'react'
+import { Space, SpaceMember } from '@ably-labs/spaces'
+import CursorSvg from './CursorSvg'
+import useCursor from './use-cursor'
+
+// 💡 This component is used to render the cursor of the user
+const YourCursor = ({
+  user,
+  space,
+}: {
+  user: SpaceMember | undefined
+  space: Space | undefined
+}) => {
+  const [cursorPosition, setCursorPosition] = useState<{
+    [key: string]: number
+  }>({ left: 100, top: 100 })
+
+  const handleSelfCursorMove = useCursor(setCursorPosition, space)
+
+  return (
+    <div
+      className="absolute"
+      onMouseMove={(e) => handleSelfCursorMove(e)}
+      style={{
+        top: `${cursorPosition.top + 30}px`,
+        left: `${cursorPosition.left}px`,
+      }}
+    >
+      <CursorSvg
+        startColor={user?.profileData.userColors.split(' ')[2]}
+        endColor={user?.profileData.userColors.split(' ')[3]}
+        id={user?.connectionId}
+      />
+      <div
+        className={`px-4 py-2 m-2 bg-gradient-to-r ${
+          user?.profileData.userColors.split(' ')[0]
+        } ${
+          user?.profileData.userColors.split(' ')[1]
+        } rounded-full text-sm text-white`}
+      >
+        {user?.profileData.name} (You)
```

Comment:
As above, we should not use `user?` here, this will just create text like `undefined (You)`